wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,324 @@
+3. [osu! forum comment (Beatmap Graveyard) [peppy] - "Best of No.1 Hits - Lemon Tree"](https://osu.ppy.sh/community/forums/topics/95725?start=105679)
+4. [osu! beatmap - "Best of No.1 Hits - Lemon Tree" (MillhioreF)](https://osu.ppy.sh/beatmapsets/57878#osu/174267)
+
+---
+
+## 2010
+
+### touhosu!
+
+For the 2010 April Fools' prank, the osu! website along with the in-game main menu screen was changed to theme with characters and references from the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (shown below). The changes included the addition of the character Marisa Kirisame and a display of differently-coloured butterflies in a circular pattern across the main menu and website, along with the swapping of the name "osu!" to "touhosu!" in certain areas across the website (shown below) \[1,2,3\].
+
+![](img/2010-touhosu-main-screen.jpg "Screenshot of a user's main menu screen in osu! on 1 April 2010")
+
+![](img/2010-touhosu-website.jpg "Screenshot of the osu! website on 1 April 2009")
+
+The joke largely stems from a [long-standing feature request](https://osu.ppy.sh/community/forums/topics/19307) to create a [game mode](/wiki/Game_mode) based off of the currently-existing [osu!catch](/wiki/Game_mode/osu!catch) game mode, and the core gameplay from the Touhou Project \[4\].
+
+There was also a report at the time of [Ephemeral](https://osu.ppy.sh/users/102335) jokingly remarking that buying an osu!supporter tag would show a naked Marisa Kirisame on the main menu screen instead of a fully clothed one, however this claim was only a joke and was quickly disproven by others \[5\].
+
+#### References
+
+1. [April Fools' Day On The Web entry - "Changed osu! to touhousu! throughout the website as well as the game."](http://aprilfoolsdayontheweb.com/joke/8120/?size=1 "April Fools' Day On The Web")
+2. [Discord message (osu!dev) [Nivalyx#9577] - 27 January 2021 (05:06 UTC)](https://discord.com/channels/188630481301012481/218677502141399041/804215894762848296 "Discord")
+3. [osu! forum thread (Resolved Issues) [rcmero] - "touhousu! - April Fools joke? [Resolved]"](https://osu.ppy.sh/community/forums/topics/27612)
+4. [osu! wiki article - "Touhosu!"](/wiki/Glossary/Touhosu!)
+5. [osu! forum thread (General Discussion) [rulingvenus] - "Naked Marisa????"](https://osu.ppy.sh/community/forums/topics/27531)
+
+---
+
+## 2011
+
+### osu!core
+
+"osu!core" is the name given to osu!'s 2011 April Fools' day prank. The prank caused every single [beatmap](/wiki/Beatmap)'s audio to be pitched and sped up in the style of [Nightcore](https://en.wikipedia.org/wiki/Nightcore "Wikipedia") remixes. Although just an April Fools' day prank, it later became a reality when the [Nightcore mod](/wiki/Game_modifier/Nightcore) was introduced as an actual playable [game modifier](/wiki/Game_modifier) in osu! later on \[1,2,3,4\].
+
+#### References
```

Comment:
Instead of creating seperate references here, use one big references section at the bottom. We want everyone to focus on the article content firsthand.